src/Storages/StorageReplicatedMergeTree.h

```diff
@@ -329,6 +330,7 @@ class StorageReplicatedMergeTree final : public shared_ptr_helper<StorageReplica
     ReplicatedMergeTreeQueue queue;
     std::atomic<time_t> last_queue_update_start_time{0};
     std::atomic<time_t> last_queue_update_finish_time{0};
+    MultiVersion<String> last_queue_update_exception;
```

Comment:
`MultiVersion` is not intended to be something like `std::atomic<AnyClass>`. Why not just string with mutex (or even reuse some mutex...) + two methods?